_includes/portfolio.html

```diff
@@ -2,7 +2,7 @@
     <div class="portfolio-centered">
         {% if include.title %}<h3>{{ include.title }}</h3>{% endif %}
         <div class="recentitems portfolio">
-            {% for project in site.categories['project'] limit:include.limit %}
+            {% for project in site.projects limit:include.limit %}
```

Comment:
This loses the date-ordering provided by `posts` and always returns an A->Z sorted list. If projects follow `_projects/date-title.md` format then the projects with the earliest date are returned first. Hence, _new projects will not appear on top_.

This can be fixed by replacing this line with
```
{% assign projects = site.projects | reverse %}
{for project in projects limit:include.limit %}
```